### PR TITLE
chore: add theme setting

### PR DESCRIPTION
**Pull request by GreyXor**
**Problem**
no theme setting in api reference

**Explanation**
This happens because :shrug: 

**Solution**
With this PR theme is here again

### CODE REVIEW

**Review by marclave — APPROVED** [review]
nice work @GreyXor ✨

**Comment by cameronrohani** [comment]
This is great!

File: packages/api-reference/README.md [hunk 1]
```diff
@@ -285,3 +286,16 @@ By default we’re using Inter and JetBrains Mono, served by Google Fonts. If yo
   withDefaultFonts: false
 } />
 ```
+
+#### theme?: string
+
+You don’t like the color scheme? We’ve prepared some themes for you:
+
+Can be one of:  **alternate**, **default**, **moon**, **purple**, **solarized**, **bluePlanet**, **saturn**, **kepler**, **mars**, **deepSpace**, **none**
```
Comment (antlio) [inline packages/api-reference/README.md:294]: should we strip that space? unless it was intentional

```suggestion
Can be one of: **alternate**, **default**, **moon**, **purple**, **solarized**, **bluePlanet**, **saturn**, **kepler**, **mars**, **deepSpace**, **none**
```

**Comment by GreyXor** [comment]
I did run `pnpm lint:fix`

**Comment by marclave** [comment]
> I did run `pnpm lint:fix`

sorry i always do this haha i meant `pnpm format`

i ran it and pushed it up !

**Pull request merged**